src/actions/measurements.js

```diff
@@ -130,3 +179,21 @@ export const changeMeasurementsCollection = (newCollectionKey) => (dispatch, get
     controls: getCollectionDisplayControls(controls, collectionToDisplay)
   });
 };
+
+/*
+ * The filter actions below will create a copy of `controls.measurementsFilters`
+ * then clone the nested Map to avoid changing the redux state in place.
+ * Tried to use lodash.cloneDeep(), but it did not work for the nested Map
+ * - Jover, 19 January 2022
+ */
+export const toggleSingleFilter = (field, value, active) => (dispatch, getState) => {
```

Comment:
Could this be renamed to `applyMeasurementFilter`?

Reply:
Resolved in https://github.com/nextstrain/auspice/pull/1452/commits/12dc1ba8c002dc5e280b7a56fc869b5fbe94ab8e